test/unit/init.js

```diff
@@ -153,16 +153,12 @@ describe('Instantiation', function () {
       skate(id, {
         type: skate.types.CLASS,
         created: function () {
-          idsToCheck[id] = true;
+          idsToCheck.push(id);
         }
       });
     });
 
     skate.init(div);
-
-    idsToSkate.forEach(function (id) {
-      idsToCheck[id].should.equal(true);
-    });
```

Comment:
I should probably actually remove this test. It's hard to grok and it's not testing anything useful. The original problem was specific to firefox and existed when you had a component called `watch`. This isn't a problem if using the recommended spec'd naming convention which we're recommending moving forward.